app/views/moves/_form.html.slim

```diff
@@ -8,7 +8,31 @@
     .form-group
       label.form-label From
       = from_establishment&.name
-    = f.text_field :to
+
+    .radio-concertina
+      .form-group
+        fieldset
+          legend
+            span.form-label-bold To
+
+          - ::Establishment::ESTABLISHMENT_TYPES.each do |option|
```

Comment:
Does it need the starting :: ?
::Establishment::ESTABLISHMENT_TYPES -> Establishment::ESTABLISHMENT_TYPES

Reply:
Yes - it failed to load without it